src/Common/src/System/Security/Cryptography/Asn1Reader/AsnCharacterStringEncodings.cs

```diff
@@ -153,6 +156,16 @@ internal VisibleStringEncoding()
         }
     }
 
+    internal class NumericStringEncoding : RestrictedAsciiStringEncoding
+    {
+        // T-REC-X.680-201508 sec 41.2 (Table 9)
+        // 0, 1, ... 9 + space
+        internal NumericStringEncoding()
+            : base("0123456789 ")
```

Comment:
haven't read the spec but presumably negative numbers are not allowed?

Reply:
Nope. 

>  41.2 Table 9 lists the characters which can appear in the NumericString type and NumericString character abstract syntax.
Table 9 – NumericString

Name | Graphic
--------|----------
Digits | 0, 1, ... 9
Space | (space)

> 41.3 The following object identifier,...